generate-wasm-examples/generate_wasm_examples.sh

```diff
@@ -42,11 +42,7 @@ add_category()
         # Patch generated JS to allow to inject custom `fetch` with loading feedback.
         # See: https://github.com/bevyengine/bevy-website/pull/355
         sed -i.bak \
-          -e 's/function init(input) {/function init(customFetch, input) { customFetch = customFetch || fetch;/' \
-          -e 's/input = fetch(/input = customFetch(/' \
```

Comment:
Be aware, I think there were to different calls to `fetch`:
- `input = fetch(` => for `.wasm` file?
- `getObject(arg0).fetch(` => for assets?

With this change-set I think only the assets loading progress will show. If you slow down the connection via "network" panel you should be able to notice it.

Reply:
Ah, good catch, yeah it didn't show up locally but I did notice a small time without a loading bar.